docs/customizing-evaluation-outputs.md

```diff
@@ -0,0 +1,55 @@
+# Template BPA Customizing Evaluation Outputs
+<a name="note"></a>***Note**: The ARM Template BPA is currently in development. All features that have yet to be implemented have been flagged with an asterisk [\*].*
+
+## Overview
+Template BPA customization is authored in JSON.  Custom evaluation settings are written in a separate configuration and used on tool execution. More details on how to run Template BPA with a configuration file can be found [here](https://github.com/Azure/template-analyzer#using-the-template-bpa). 
+
+## Template BPA Rule Object
+Here are the fields that make up a custom configuration file:
+```javascript
+{
+    "exclusions": { � }, // Parameters that are excluded from the BPA execution. More details below.
+    "inclusions": { � }, // Only parameters that will be included in the BPA execution. More details below.
+    "output": { � }, // How the BPA execution output will be displayed to the user. More details below.
+    "severityReMapping": { � } // Tuple of Id and new severity value. Can change a rule's severity.
+}
+```
+
+### Guidelines for exclusions object
+The `exclusions` object is compromised of the following optional properties:
+```javascript
+"exclusions": {
+    "severity": [int], // Means don't include rules with severities that are in this list in the BPA execution
+    "ids": ["Id"] // Means don't include rules that are in this list in the BPA execution
+}
+```
+If the inclusions object contains values, then the exclusions object will be ignored. 
+
+### Guidelines for inclusions object
+The `inclusions` object is compromised of the following optional properties:
+```javascript
+"inclusions": {
+    "ids": ["Id"] // Means only include rules that are in this list in the BPA execution
+}
```

Comment:
It can be. At the time, it seemed redundant to have it be in both as there are only three severity options. Should I add it in?

Reply:
Ah, you mean if someone wants to only include '3', they can just exclude '1' and '2'?  That could work.  If it's easy enough though, it might be more user-friendly to just support it in both ways.